src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (result[robot[key]]) {
+      return null;
+    } else {
+      result[robot[key]] = key;
+    }
```

Comment:
when you return something in `if` statement you might not use else

```suggestion
    if (result[robot[key]]) {
      return null;
    }

    result[robot[key]] = key;

```

Reply:
Forgot, sorry.